.ci/deps.apt.sh

```diff
@@ -10,6 +10,7 @@ set -x
 export DEBIAN_FRONTEND=noninteractive
 
 deps="libclang1-3.4 astyle indent mono-mcs chktex r-base julia golang-go luarocks verilator cppcheck flawfinder devscripts mercurial"
+deps="libclang1-3.4 indent mono-mcs chktex r-base julia golang-go luarocks verilator cppcheck flawfinder devscripts pychecker"
```

Comment:
Just a duplicate of above declared `deps` variable with some new dependencies.
Add new dependencies in the above declared `deps` variable instead of overwriting it again!